src/components/accordion/index.md.njk

```diff
@@ -118,6 +118,8 @@ The team made sure the component is accessible, for example that users can inter
 
 ### Known issues and gaps
 
-Using summary text is almost certain to make a section heading’s button text too long for screen reader users.
+An accordion button that includes a summary line is likely to be too long, according to general best practice for button text.
```

Comment:
I would change this to 
> that includes a **long** summary line is likely to be too long

I think a few word summary is more likely to be fine (even though perhaps still not ideal) but a really long summary line could be an issue.

Reply:
The 'a few short words' amendment is great 👍   I'm a bit unclear on we we mean by 'Also adding the reason to be a bit clearer.'

Reply:
Thanks! 

Oh no, I accidentally out that into the code!

I was just saying I'd added the 'screen readers' line to give a reason why it's too long. It might not be necessary though?

Reply:
@calvin-lau-sig7 I actually really like the 'screen readers' addition, I think it's useful in setting out the context.